plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -159,12 +159,17 @@ class NewCreateServiceModalForm extends Component {
       return true;
     }
 
+    const isBaseConfigChanged = this.state.baseConfig !== nextState.baseConfig;
```

Comment:
Thanks @mlunoe - I won't get a chance to do this for a couple of days so I appreciate the help if someone can make the changes. Instead of `isBaseConfigChanged` it should be `didBaseConfigChange`. We are mixing tenses and it's confusing.